mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -435,7 +435,6 @@ def __init__(
         PrescribedFluidBoundary.__init__(
             self,
             boundary_state_func=self.get_external_state,
-            boundary_grad_av_func=self.get_external_grad_av,
```

Comment:
@majosm: Due to upstream changes in PhysicalAV #735 , AV funcs no longer in the fluid boundary constructs.  We'll remove the old AV as soon as we get it weeded out of our stuff.